multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2257,12 +2264,17 @@ def _match_queries_and_candidates(
 
     def predict(
         self,
-        data: Union[pd.DataFrame, dict, list],
+        data: Union[pd.DataFrame, dict, list, str],
         candidate_data: Optional[Union[pd.DataFrame, dict, list]] = None,
         id_mappings: Optional[Union[Dict[str, Dict], Dict[str, pd.Series]]] = None,
         as_pandas: Optional[bool] = None,
         realtime: Optional[bool] = None,
         seed: Optional[int] = 123,
+        save_results: Optional[bool] = False,
```

Comment:
Do we need to add `save_results` and `result_path` for `evaluate()`?

Reply:
For evaluate, the groundtruth is available so I was thinking user don't actually need those results.

Reply:
They might need it for error analysis, e.g., they can use the TIDE toolbox (https://dbolya.github.io/tide/) to see where their predictions are off. But we can mark a TODO here.